ocaml/quicktest/quicktest_storage.ml

```diff
@@ -616,12 +616,11 @@ let foreach_sr session_id sr =
   | _ ->
     failed test "Multiple plugins with the same type detected"
 
-let go s =
+let go s use_default_sr =
   let test = make_test "Listing available Storage Repositories" 0 in
   start test;
-  let srs = list_srs s in
+  let srs = list_srs s use_default_sr in
   debug test (Printf.sprintf "Found %d SRs" (List.length srs));
   success test;
-  if !only_sr_name = None then
```

Comment:
I think the reason for skipping the ISO test was that the ISO test was failing on our branch. If you tried this on your box and all tests pass, then deleting this line is correct.

Reply:
Yes, it ran without problems on my dtbox